tests/test_pending_points.py

```diff
@@ -0,0 +1,145 @@
+"""Tests pending points mechanism."""
+
+import warnings
+
+import pandas as pd
+import pytest
+from pytest import param
+
+from baybe.acquisition.base import AcquisitionFunction
+from baybe.exceptions import IncompatibleAcquisitionFunctionError, UnusedObjectWarning
+from baybe.recommenders import (
+    BotorchRecommender,
+    FPSRecommender,
+    GaussianMixtureClusteringRecommender,
+    KMeansClusteringRecommender,
+    NaiveHybridSpaceRecommender,
+    PAMClusteringRecommender,
+    TwoPhaseMetaRecommender,
+)
+from baybe.utils.basic import get_subclasses
+from baybe.utils.dataframe import add_fake_results, add_parameter_noise
+from baybe.utils.random import temporary_seed
+
+_discrete_params = ["Categorical_1", "Switch_1", "Num_disc_1"]
+_continuous_params = ["Conti_finite1", "Conti_finite2", "Conti_finite3"]
+_hybrid_params = ["Categorical_1", "Num_disc_1", "Conti_finite1", "Conti_finite2"]
+
+# Repeated recommendations explicitly need to be allowed or the potential overlap will
```

Comment:
This makes me think - isn't this something that we should also test/check/mention/verify.... at other places as well?

Reply:
Please elaborate as I dont know what you mean. This test explicitly needs to allow repeated recommendations.

Reply:
What I mean here is that we should mention that we should communicate that "if you use pending points, make sure that you allow repeated recommendations" resp. that this is something that should be checked/verified.

Reply:
you dont have to allow repeated recommendations with pending points, is there anything particular that made you think that?
im merely suing the fact that a pending point should not be recommended, this is how the text is constructed, but it doesnt mean in any way that users need to set that setting

Reply:
@AVHopp requiring your response or resolution